Add references to Ecosystem Implementation Guidelines section (#312)
vcstuff/oid4vc-haip-sd-jwt-vc: openid4vc-high-assurance-interoperability-profile-1_0.md

## Ecosystem Implementation Considerations

This document intentionally leaves certain extensions for ecosystems to define, in order to enable broad compatibility across differing or even conflicting requirements. Below are the extension points listed in this specification:

> - Whether to adopt a presentation profile, issuance profile, or both (see (#scope))

Reviewer: ```suggestion
- Which flow(s) to adopt: presentation, issuance, or both (see (#scope))
```

- Whether to use the W3C Digital Credentials API, Redirects with custom URL schemes and/or Redirects with claimed `https` scheme URIs for presentation (see (#scope))
- Which Credential format to support across issuance and presentation (see (#scope))
- Whether to use Signed Issuer Metadata or not (see (#issuer-metadata))

> - How to make a Credential Offer to the Wallet (see (#credential-offer))

Reviewer: ```suggestion
- How to make a Credential Offer available to the Wallet (see (#credential-offer))
```

> - Which key attestation format to use (see (#key-attestation))

Reviewer 2: > I struggled with this. I didn't find there was consistent capitalisation in the spec at this point in time. Happy for this to change, but given that in other places it was done, I'm not sure this was intentional

The intent was to have all introduced terminology (such as Wallet Attestation, Key Attestation, etc.) capitalised - that is what we tried to do for all 3 specs.
